tcms/bugs/api.py

```diff
@@ -57,6 +57,22 @@ def remove_tag(bug_id, tag):
     )
 
 
+@permissions_required('bugs.delete_bug')
+@rpc_method(name='Bug.delete_bug')
+def delete_bug(bug_id):
```

Comment:
method & perms are called remove, see other API methods. And b/c you've used the wrong perm initially you will need to add a test for that, to make sure we use the correct one.